pkg/logs/launchers/file/launcher.go

```diff
@@ -294,6 +334,8 @@ func (s *Launcher) launchTailers(source *sources.LogSource) {
 			continue
 		}
 
+		s.addedSourceTailers = append(s.addedSourceTailers, file)
```

Comment:
Maybe we only want to do this if `startNewTailer` returns true? (i'd double check)

Reply:
That makes sense, instead of just appending all files.